packages/desktop-client/src/components/reports/chart-theme.ts

```diff
@@ -41,7 +75,82 @@ const axisBaseStyles = {
   tickLabels: baseLabelStyles,
 };
 
-export const chartTheme = {
+type ChartTheme = {
+  colors: {
+    [key: string]: string;
+  };
+  area: {
+    style: {
+      labels: LabelStyles;
+      data: {
+        stroke: string;
+        strokeWidth: number;
+        strokeLinejoin: string;
+        strokeLinecap: string;
+      };
+    };
+  };
+  axis: {
+    style: AxisBaseStyles;
+  };
+  dependentAxis: {
+    style: {
+      grid: {
+        stroke: string;
+        strokeDasharray: string;
+      };
+      tickLabels: {
+        padding: number;
+      };
+    };
+  };
+  independentAxis: {
+    style: {
+      axis: {
+        stroke: string;
+      };
+      tickLabels: {
+        padding: number;
+      };
+    };
+  };
+  bar: {
+    style: {
+      labels: LabelStyles;
+      data: {
+        fill: string;
+        stroke: string;
+      };
+    };
+  };
+  line: {
+    style: {
+      labels: LabelStyles;
+      data: {
+        fill: string;
+        stroke: string;
+        strokeWidth: number;
+        strokeLinejoin: string;
+        strokeLinecap: string;
+      };
+    };
+  };
+  voronoi: {
+    style: {
+      labels: LabelStyles;
+    };
+  };
+  chart: {
+    padding: {
+      top: number;
+      left: number;
+      right: number;
+      bottom: number;
+    };
+  };
+};
+
+export const chartTheme: ChartTheme = {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏and here
```suggestion
export const chartTheme = {
```

Reply:
Removed it